src/spartan_app.erl

```diff
@@ -125,6 +129,9 @@ load_json_config(FileBin) ->
 process_config_tuple({<<"upstream_resolvers">>, UpstreamResolvers}) ->
     UpstreamIPsAndPorts = lists:map(fun (Resolver) -> parse_ipv4_address_with_port(Resolver, 53) end, UpstreamResolvers),
     application:set_env(?APP, upstream_resolvers, UpstreamIPsAndPorts);
+process_config_tuple({<<"forward_zones">>, Upstreams0}) ->
+    Upstreams1 = lists:foldl(fun parse_upstream/2, #{}, Upstreams0),
```

Comment:
My personal preference is to use maps:new() instead of #{} that way we safeguard ourselves from internal changes to maps module